example/esy.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "@opam/atd": "*",
+    "@opam/atdgen": "*"
+  },
+  "resolutions": {
+    "@opam/atd": "github:ahrefs/atd:atd.opam#6a1f166",
```

Comment:
Hm, I just read this in main readme:

> The version 1.1.0 of bs-atdgen-codec-runtime is known to work with the version 8c08659 of atdgen.

Should I point to this same version? Alternatively, what should be done / tested to safely bump this dependency?

Reply:
now that atdgen has been released, the latest version should be usable directly

Reply:
now that atdgen has been released we can used the latest version instead of a specific commit